vars/createGithubIssue.groovy

```diff
@@ -1,6 +1,14 @@
 def call(Map args = [:]){
     def failureMessages = args.message
     List<String> failedComponents = []
+    def manifestPath = "${WORKSPACE}/manifests"
+
+    if (fileExists(manifestPath)) {
+        println("Manifest path exists, not checking out git repo")
+    } else {
+        println("Manifest path does not exists, checking out git repo")
+        git url: 'https://github.com/opensearch-project/opensearch-build.git', branch: 'main'
```

Comment:
Since we are cloning again, can you clean up the workspace after your execution is done? Just add postCleanup() at the end.

Reply:
Looks like there is a postCleanup() after your script in the jenkinsFile https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/distribution-build.jenkinsfile#L643